templates/includes/sidebar.html

```diff
@@ -1,4 +1,12 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    User:
+    <a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a>
```

Comment:
Ensure that the `user.id` corresponds to a driver's primary key in the `driver-detail` URL. If `user.id` does not directly map to a driver's ID, this link may not work as intended.